README.md

```diff
@@ -1,6 +1,5 @@
 # scala-parser-combinators
 
-[<img src="https://img.shields.io/travis/scala/scala-parser-combinators.svg"/>](https://travis-ci.org/scala/scala-parser-combinators)
```

Comment:
GitHub actions also has badges, see https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge. The badge could link to https://github.com/scala/scala-parser-combinators/actions.

Reply:
I'm not opposed if someone wants to add this in a separate PR.